modules/videoio/include/opencv2/videoio/videoio_c.h

```diff
@@ -240,27 +240,147 @@ enum
     CV_CAP_PROP_PVAPI_PIXELFORMAT           = 306, // Pixel format
 
     // Properties of cameras available through XIMEA SDK interface
-    CV_CAP_PROP_XI_DOWNSAMPLING  = 400,      // Change image resolution by binning or skipping.
```

Comment:
master now represents updates to the stable 3.0 release, where we want to preserve binary compatibility. Changing numerical values of the enumeration constants obviously breaks binary compatibility. Could you please retain the original list as-is and simply add the new values in the end?